docker/develop/Dockerfile

```diff
@@ -185,5 +185,13 @@ RUN git clone git://sourceware.org/git/valgrind.git /tmp/valgrind; \
     make -C /tmp/valgrind install; \
     rm -rf /tmp/valgrind
 
+# install ed25519
+RUN git clone git://github.com/warchant/ed25519.git /tmp/ed25519; \
+    (cd /tmp/ed25519 ; git checkout 0d31b082e0f0d4144fe598fb0f2a120e36de2a8f); \
+    cmake -DCMAKE_BUILD_TYPE=Debug -DTESTING=OFF -H/tmp/ed25519 -B/tmp/ed25519/build; \
```

Comment:
Why not build in release?

Reply:
Otherwise you will not be able to debug in `develop` environment. 

Well, we actually need 2 environments: one is for Release builds, one is for Debug builds.
But now we have just one.